pkg/ssh/ssh.go

```diff
@@ -198,11 +198,22 @@ func (d *timeoutDialer) Dial(network, addr string, config *ssh.ClientConfig) (*s
 // host as specific user, along with any SSH-level error.
 // If user=="", it will default (like SSH) to os.Getenv("USER")
 func RunSSHCommand(cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
-	return runSSHCommand(realTimeoutDialer, cmd, user, host, signer, true)
+	return runSSHCommand(realTimeoutDialer, cmd, user, host, nil, signer, true)
+}
+
+// RunSSHCommandWithStdin sends input as stdin and returns the stdout, stderr, and exit code from running cmd on
+// host as specific user, along with any SSH-level error.
+// If user=="", it will default (like SSH) to os.Getenv("USER")
+func RunSSHCommandWithStdin(input, cmd, user, host string, signer ssh.Signer) (string, string, int, error) {
+	var inputreader io.Reader
+	if input != "" {
+		inputreader = strings.NewReader(input)
+	}
+	return runSSHCommand(realTimeoutDialer, cmd, user, host, inputreader, signer, true)
 }
 
 // Internal implementation of runSSHCommand, for testing
-func runSSHCommand(dialer sshDialer, cmd, user, host string, signer ssh.Signer, retry bool) (string, string, int, error) {
+func runSSHCommand(dialer sshDialer, cmd, user, host string, input io.Reader, signer ssh.Signer, retry bool) (string, string, int, error) {
```

Comment:
You'll need to update pkg/ssh/ssh_test.go as well, this is why the bazel and verify tests are failing:

```
W0720 17:30:18.719] Use --strategy=GoCompile=standalone to disable sandboxing for the failing actions.
I0720 17:30:18.819] pkg/ssh/ssh_test.go:307: not enough arguments in call to runSSHCommand
I0720 17:30:18.820] 	have (*mockSSHDialer, string, string, string, ssh.Signer, bool)
I0720 17:30:18.821] 	want (sshDialer, string, string, string, io.Reader, ssh.Signer, bool)
W0720 17:30:20.296] ____Building complete.
```